### DISCUSSION TITLE
Investigate the possibility of collaborative dashboards using CRDTs

### DISCUSSION THREAD

**Discussion by kaydelaney** [grafana/grafana#38249] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
The ability to collaborate in real-time with other users on a single Grafana dashboard.

**Why is this needed**:
Currently we don't have a great solution for when two or more people want to make changes to the same dashboard
concurrently. We have some safeguards to prevent people's work from getting overwritten, but no easy way to collaborate together.
One solution to this problem could be to implement "collaborative dashboards" (think google docs, but for Grafana dashboards). Historically collaborative tools like google docs have been implemented using [operational transformations](https://en.wikipedia.org/wiki/Operational_transformation), but a more recent approach is to use [conflict-free replicated data type (CRDTs)](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type).
There are several libraries to help facilitate this, including [Yjs](https://github.com/yjs/yjs) and [Automerge](https://github.com/automerge/automerge), among others.

**Comment by ryantxu** [comment]
Also with grafana live... We could broadcast all changes, not just saved ones!

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
